writing-adapters.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Adapter Writing Guide
+description: This guide provides a step-by-step approach to creating adapters for the Truf Network (TN). Adapters facilitate the ingestion of data from various sources into TN, leveraging the SDK and Prefect for efficient data management.
+---
+
+# Developing Adapters for the Truf Network
+
+## Prerequisites
+
+- **Programming Knowledge**: Proficiency in Python.

```

Comment:
@MicBun can adapters be written in other languages?

Reply:
Yes, it is actually possible to be written in Go and Js. The example is given in Python. I will revise it.